auto_py_to_exe/web/js/i18n.js

```diff
@@ -1,5 +1,5 @@
 // Constants used in translation map
-const applicationName = 'auto-py-to-exe';
+const applicationName = "auto-py-to-exe";
```

Comment:
Can you run `npx prettier --write .` from the root directory so the config in `.prettierrc` is used?